framework/db/ActiveRecord.php

```diff
@@ -179,7 +179,7 @@ protected static function findByCondition($condition)
             if (isset($primaryKey[0])) {
                 $pk = $primaryKey[0];
                 if (!empty($query->join) || !empty($query->joinWith)) {
-                    $pk = static::tableName() . '.' . $pk;
+                    $pk = ($query->from ? key($query->from) : static::tableName()) . '.' . $pk;
```

Comment:
this is not correct, `$query->from` could be `['sometable']` where the key would be 0.
This should use similar approach as `ActiveQuery::getTableNameAndAlias()` but that method is protected right now.

Reply:
Should I create new method like `getAlias()`? Or I can just change `protected` to `public`, but it's doubtful. What's better solution?

Reply:
ActiveQuery should not have public methods other than those building the query, so a private function for getting the alias should be used. It's okay to duplicate some code from ActiveQuery. I don't see a better way to solve it right now.